project/plugins.sbt

```diff
@@ -3,3 +3,5 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.17.0")
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.5.5")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.7")
 addSbtPlugin("io.circe" % "sbt-circe-org" % "0.4.3")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.13.0")
```

Comment:
i think we should update these in sbt-circe-org instead, se https://github.com/circe/sbt-circe-org/pull/75